### DISCUSSION TITLE
RemoveApps parameter

### DISCUSSION THREAD

**Discussion by ZFranz** [Raphire/Win11Debloat#98] [Q&A]
I'm trying to understand the code, but I can't understand where that the function removeApps get its list with the default switch, any kind soul that would explain that?

**Comment by Raphire** [comment]
I think you may be confusing the `RemoveApps` parameter with the function that has the same name. 

When you use the `RemoveApps` parameter it will actually call the `RemoveAppsFromFile` function with the path to the appslist.txt file. See here for reference: https://github.com/Raphire/Win11Debloat/blob/69590c55e203f887c2873b225f6fc36ca8b37884/Win11Debloat.ps1#L1213

The `RemoveAppsFromFile` function parses the appslist.txt file into a list of apps that are then removed using the RemoveApps function.
